src/main/resources/env/init_db_schema.json

```diff
@@ -137,6 +140,7 @@
     "provider": {
       "name": "EMC"
     },
+    "license": "BSD",
```

Comment:
These values are taken from recodid and early_cause cohorts and reflect real values. So we can not add "license" filed in any of these cohorts.

Reply:
ok, just added for testing , will remove it.